chain/messagepool/repub.go

```diff
@@ -91,8 +93,18 @@ func (mp *MessagePool) republishPendingMessages() error {
 
 		// does it fit in a block?
 		if chain.gasLimit <= gasLimit {
-			gasLimit -= chain.gasLimit
-			msgs = append(msgs, chain.msgs...)
+			// check the baseFee lower bound -- only republish messages that can be included in the chain
+			// within the next 20 blocks.
+			for _, m := range chain.msgs {
+				if m.Message.GasFeeCap.LessThan(baseFeeLowerBound) {
```

Comment:
Instead of doing this, pass the `bsaeFeeLowerBound` to `createMessageChains`

Reply:
probably the right thing to do, as it can apply to everything.

Reply:
This logic has to stay because otherwise messages will get rejected.

Reply:
done.